vernac/obligations.ml

```diff
@@ -924,6 +928,22 @@ in
       ignore (auto (Some prg.prg_name) None deps)
   end
 
+type obligation_reference =
+  | OblNumber of int
+  | OblName of Id.t
+
+let get_obligation_index prg obl =
+  let obls, rem = prg.prg_obligations in
+  match obl with
+  | OblNumber user_num ->
+     let num = pred user_num in
+     if num < Array.length obls then num
+     else error (sprintf "Unknown obligation number %i" (succ num))
```

Comment:
Another command does that, Obligations, which I guess the user would call anyway to see which obligations are there